README.md

```diff
@@ -33,12 +33,16 @@ The following games/languages are currently supported:
 | German   | ❌ | ❌ | ❌ | ✔️ | ❌ |
 | Japanese | ❌ | ❌ | ❌ | ❌ | ❌ |
 
-We'd ideally like to support all non-English versions if we can, see [here](https://github.com/besteon/Ironmon-Tracker/issues/62) for progress updates if you're curious.
+We'd ideally like to support all non-English versions if we can, progress updates can be found [here](https://github.com/besteon/Ironmon-Tracker/issues/62).
 
 ## Installation
 
-1. **Get Bizhawk**: If you haven't used BizHawk before, [download the emulator](https://tasvideos.org/BizHawk/ReleaseHistory) (v2.8 or higher)
-   - If you are on windows, make sure to also download and run the [prereq installer](https://github.com/TASEmulators/BizHawk-Prereqs/releases) first.
+1. **Get a Supported Emulator**:
+   - We recommend using the Bizhawk emulator (Windows/Linux only).
+	   - [Download Bizhawk](https://tasvideos.org/BizHawk/ReleaseHistory) (v2.8 or higher)
+		- If you are on windows, make sure to also download and run the [prereq installer](https://github.com/TASEmulators/BizHawk-Prereqs/releases) first.
```

Comment:
Could perhaps also add the recommendation to use 2.9 on linux for full functionality here too?

Reply:
Remind me to update this for 7.1.1. I think its okay for this release. 2.9 isn't even out yet technically